packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -36,7 +36,7 @@ class InnerBlocks extends Component {
 	}
 
 	componentDidMount() {
-		const { templateLock, block } = this.props;
+		const { block, templateLock, blocks, replaceInnerBlocks } = this.props;
```

Comment:
Would it make sense to mark blocks property as experimental until FSE is stable?
Blocks property acts exactly like a template -- we prefill an InnerBlocks area, future changes to the property are not taken into consideration. The only difference compared to templates is that blocks is a normal block structure with attributes cliendIds etc while the template uses a specific template structure.
I wonder if the need to have this type of prefilling will be common that worths being part of InnerBlocks or if it is something specific that should be part of the template part bock itself.

Reply:
A lot of blocks will use it: Post Content, Reusable, Query, basically any blocks that save their children or a representation of them somewhere other than the current post.

I'm comfortable with it not being experimental, but we can change it if you have reservations about the API.

Reply:
> A lot of blocks will use it: Post Content, Reusable, Query, basically any blocks that save their children or a representation of them somewhere other than the current post.

> I'm comfortable with it not being experimental, but we can change it if you have reservations about the API.

Hi @epiqueras,

The blocks that use this API don't seem "normal" blocks and have a very specific need I'm not sure if this API is useful for normal InnerBlocks use cases. I guess in the future we may have an alternative abstraction to InnerBlocks used by post content, reusable, etc... specific for these blocks (it may use InnerBlocks and handle the replacing, etc..). For now, I think it would be safer to have an experimental API until more blocks are implemented and we have more knowledge on the direction.

Reply:
I'll make it experimental for now.